### PR TITLE
add: Blaze and Vampire slayers to st pv

### PR DESCRIPTION
**Pull request by Fix3dll**
**Slayers:**
slayer

**Heads: (Before -> After)**
beforeAfter

### CODE REVIEW

File: src/main/kotlin/gg/skytils/skytilsmod/gui/profile/components/ItemComponent.kt [hunk 1]
```diff
@@ -38,7 +38,7 @@ class ItemComponent(val state: State<ItemStack>) : UIComponent() {
         super.draw(matrixStack)
         matrixStack.push()
         matrixStack.translate(getLeft(), getTop(), 100f)
-        matrixStack.scale(getWidth() / 16f, getHeight() / 16f, 0f)
```
Comment (Sychic) [inline src/main/kotlin/gg/skytils/skytilsmod/gui/profile/components/ItemComponent.kt:41]: Is this change necessary? It should adjust the skull to be the appropriate size for an arbitrarily sized `ItemComponent`

**Review by Sychic — APPROVED** [review]
Just one question, otherwise LGTM.

File: src/main/kotlin/gg/skytils/skytilsmod/gui/profile/components/ItemComponent.kt (hunk 1 above)
Comment (Fix3dll) [reply src/main/kotlin/gg/skytils/skytilsmod/gui/profile/components/ItemComponent.kt:41]: This scaling breaking skull textures. I didnt see any problem in scaling when i tried it without this. (as seen in the images above) Thats why i didnt think there was a need to alternative scaling. Another e.g. using ItemComponent is
**SuperboomToast (Old -> New)**
toast

Comment (Sychic) [reply src/main/kotlin/gg/skytils/skytilsmod/gui/profile/components/ItemComponent.kt:41]: Those all used 16 pixel scalings, could you see if it still works with something like 20 pixels or larger like 32?

Comment (Fix3dll) [reply src/main/kotlin/gg/skytils/skytilsmod/gui/profile/components/ItemComponent.kt:41]: `matrixStack.scale(getWidth() / 32f, getHeight() / 32f, 0f)`
image
`matrixStack.scale(getWidth() / 20f, getHeight() / 20f, 0f)`
image
`matrixStack.scale(getWidth() / 4f, getHeight() / 4f, 0f)`
image
😂

Comment (Sychic) [reply src/main/kotlin/gg/skytils/skytilsmod/gui/profile/components/ItemComponent.kt:41]: Hm very interesting. I'll have to investigate this further later.
